internal/pkg/manifest/validate_env.go

```diff
@@ -39,6 +39,9 @@ func (e EnvironmentConfig) validate() error {
 	if err := e.Network.VPC.SecurityGroupConfig.validate(); err != nil {
 		return fmt.Errorf(`validate "security_group": %w`, err)
 	}
+	if err := e.CDNConfig.validate(); err != nil {
+		return fmt.Errorf(`validate "cdn config": %w`, err)
```

Comment:
the users won't know what "cdn config" means. They can only see `cdn` in the env manifest doc.
```suggestion
		return fmt.Errorf(`validate "cdn": %w`, err)
```